samples/sample_misc/wayland/include/class_wayland.h

```diff
@@ -38,7 +42,14 @@ struct ShmPool {
     unsigned size;
 };
 
-class Wayland {
+/* buffer Struct */
+struct buffer {
+   struct wl_buffer *buffer;
+   mfxFrameSurface1 *pInSurface;
+   bool done;
+};
+
```

Comment:
it's not used directly in header. Can we make typename buffer in header and declare struct in cpp file?

Reply:
it need to be use in listener_wayland.cpp under buffer_release function.

Reply:
Sorry, I didn't get your reply. Yes, it's used in cpp file, that's why I'd propose to move declaration in cpp file :)

Reply:
Ok, I already move it to declare in cpp file and typename in header. And I also make a bit code fixed changed as well.  I already push in new changed as well.